nncf/torch/graph/operator_metatypes.py

```diff
@@ -909,12 +909,20 @@ class PTModuleEmbeddingMetatype(PTModuleOperatorSubtype):
     weight_port_ids = [1]
 
 
+@PT_OPERATOR_METATYPES.register(is_subtype=True)
+class FXEmbeddingMetatype(PTModuleOperatorSubtype):
+    name = "EmbeddingOp"
+    module_to_function_names = {NamespaceTarget.ATEN: ["embedding"]}
+    hw_config_names = [HWConfigOpName.EMBEDDING]
+    weight_port_ids = [0]
+
+
```

Comment:
This metatype is ummbiguous, please move it to a `fx/operator_metatypes.py` file and do not register it in PT_OPERATOR_METATYPES

Reply:
That sounds better, should I create it in `experimental/torch/fx` or in  `torch/graph/fx`?

Reply:
`experimental/torch/fx`

Reply:
Alright I will do that